### ISSUE TITLE
Problems with Supabase Service again [see issue #95 for ongoing fix]

### ISSUE THREAD

**Issue by Ohmsnipe** [TracecatHQ/tracecat#96]
**Describe the bug**
Firstly your skript says "run in --debug mode for more information". But your parameter is "--tail" described correctly on your webpage but wrong information in skript.

I installed Supabase via .deb. and face this issue now: ( i again added a echo $output after the supabase start)

```
xxx:~/Desktop/tracecat$ bash services.sh start --env prod --tail
Environment: prod
Using command 'docker compose -f docker-compose.yaml -f docker-compose.prod.yaml'.
Starting Tracecat application setup...
No .env file detected. Running setup.
We recommend using ngrok https://ngrok.com/ to set up a static domain for your Runner URL.
Enter the Runner URL (required, e.g., https://your-ngrok-static-domain.ngrok-free.app):https://worthy-still-camel.ngrok-free.app
Enter your OpenAI API key to use AI functionality (optional, press Enter to skip):
Enter your Resend API key to use Email functionality (optional, press Enter to skip):
Generating new service key and signing secret...
Generating a Fernet encryption key for the database...
Creating new .env from .env.example...
Initializing Supabase services...
                                                                           ⡿ Sta                                                                      ⣷ Starting                     ⣻ Starting sysl                                                            ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                                                                   ⡿ Sta                                                                      ⣷ Starting             ⣻ Starting sysl                                                            ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                                                           ⡿ Sta                                                                      ⣷ Starting     ⣻ Starting sysl                                                            ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                                                   ⡿ Sta                                                                      ⣷ Starting                                                                 ⣻ Starting sysl                                                            ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                                           ⡿ Sta                                                                      ⣷ Starting                                                                 ⣻ Starting sysl                                                            ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                                   ⡿ Sta                                                                      ⣷ Starting                                                                 ⣻ Starting sysl                                                        ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                           ⡿ Sta                                                                      ⣷ Starting                                                                 ⣻ Starting sysl                                                ⣟ Starting syslog dr                                                       ⣾ Starting syslog driver.                   ⡿ Sta                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                        ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                        ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                    ⣯ Starting database.                                                            ⣻ Starting database.        ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                            ⣻ Starting database.                                                            ⣯ Starting database.⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                    ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                    ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣯ Starting database.                                                        ⣻ Starting database.                                                            ⣯ Starting database.            ⣻ Starting database.                                                            ⣯ Starting database.                                                            ⣻ Starting database.                                                ⣯ Starting database.                                                            ⣻ Starting database.    ⣯ Starting database.                                                            ⣻ Starting database.                                                            ⣟ Setting up initial                                            ⣾ Setting up ini                                                                    ⢿ Setting up        ⣯ Settin                                                                            ⣽ Se                                                    ⢿ Setting up initial schema.                            ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                                        ⣻ Settin                                        ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                            ⣟ Setting up initial                                                                ⣾ Setting up ini                        ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                            ⢿ Setting up initial schema.                                                        ⣯ Setting up initial sch        ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                        ⣻ Settin                                                                            ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                            ⣟ Setting up initial                                        ⣾ Setting up ini                                                                    ⢿ Setting up    ⣯ Settin                                                                            ⣽ Se                                                    ⢿ Setting up initial schema.                        ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                                        ⣻ Settin                                    ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                        ⣟ Setting up initial                                                                ⣾ Setting up ini                    ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                        ⢿ Setting up initial schema.                                                        ⣯ Setting up initial sch    ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                    ⣻ Settin                                                                            ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                            ⣟ Setting up initial                                    ⣾ Setting up ini                                                                    ⢿ Setting up⣯ Settin                                                                            ⣽ Se                                                    ⢿ Setting up initial schema.                    ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                                        ⣻ Settin                                ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                    ⣟ Setting up initial                                                                ⣾ Setting up ini                ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                    ⢿ Setting up initial schema.                                                        ⣯ Setting up initial sch⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                                ⣻ Settin                                                                            ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                            ⣟ Setting up initial                                ⣾ Setting up ini                                                                    ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                                    ⢿ Setting up initial schema.                ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                ⣷ Setting up                                                                        ⣻ Settin                            ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                ⣟ Setting up initial                                                                ⣾ Setting up ini            ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                ⢿ Setting up initial schema.                                                        ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                            ⣻ Settin                                                                            ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                                            ⣟ Setting up initial                            ⣾ Setting up ini                                                                    ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                                                    ⢿ Setting up initial schema.            ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                            ⣷ Setting up                                                                        ⣻ Settin                        ⣟ Se                                                    ⣷ Setting up initial schema.                                                        ⣻ Setting up initial sch                                            ⣟ Setting up initial                                                                ⣾ Setting up ini        ⢿ Setting up                                                                        ⣯ Settin                                                                            ⣽ Se                            ⢿ Setting up initial schema.                                                        ⣯ Setting up initial sch                                                            ⣽ Setting up initial                                                                ⡿ Setting up ini                                                                    ⣷ Setting up                                        ⣻ Settin                                                                            ⣟ Se                        ⣯ Setting up initial schema..Seeding data supabase/seed.                                                              ⣽ Setting up initi                                                                  ⡿ Setting up i                              ⣷ Setting                                                                           ⣻ Sett                                                                             ⣟ S                                                 ⣷ Setting up initial schema                                                         ⣻ Setting up initial sc             ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                             ⣯ Setti                                                                             ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                             ⣽ Setting up initia                                             ⡿ Setting up in                                                                     ⣷ Setting u         ⣻ Setti                                                                             ⣟ S                                                     ⣷ Setting up initial schema                             ⣻ Setting up initial sc                                                             ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                                         ⣯ Setti                                         ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                             ⣽ Setting up initia                                                                 ⡿ Setting up in                         ⣷ Setting u                                                                         ⣻ Setti                                                                             ⣟ S                                             ⣷ Setting up initial schema                                                         ⣻ Setting up initial sc         ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                         ⣯ Setti                                                                             ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                             ⣽ Setting up initia                                         ⡿ Setting up in                                                                     ⣷ Setting u     ⣻ Setti                                                                             ⣟ S                                                     ⣷ Setting up initial schema                         ⣻ Setting up initial sc                                                             ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                                         ⣯ Setti                                     ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                         ⣽ Setting up initia                                                                 ⡿ Setting up in                     ⣷ Setting u                                                                         ⣻ Setti                                                                             ⣟ S                                         ⣷ Setting up initial schema                                                         ⣻ Setting up initial sc     ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                     ⣯ Setti                                                                             ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                             ⣽ Setting up initia                                     ⡿ Setting up in                                                                     ⣷ Setting u ⣻ Setti                                                                             ⣟ S                                                     ⣷ Setting up initial schema                     ⣻ Setting up initial sc                                                             ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                                         ⣯ Setti                                 ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                     ⣽ Setting up initia                                                                 ⡿ Setting up in                 ⣷ Setting u                                                                         ⣻ Setti                                                                             ⣟ S                                     ⣷ Setting up initial schema                                                         ⣻ Setting up initial sc ⣟ Setting up initia                                                                 ⣾ Setting up in                                                                     ⢿ Setting u                                                 ⣯ Setti                                                                             ⣽ S                                                     ⢿ Setting up initial schema                                                         ⣯ Setting up initial sc                                                             ⣽ Setting up initia                                 ⡿ Setting up in                                                                     ⣷ Setting up initial schema..supabase_analytics_tracecat container logs: warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210322193905_rename_columns_for_mnesia_compatibility.exs:3: Logflare.Repo.Migrations.RenameColumnsForMnesiaCompatibility warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210729161959_subscribe_to_postgres.exs:19: Logflare.Repo.Migrations.SubscribeToPostgres warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210729161959_subscribe_to_postgres.exs:20: Logflare.Repo.Migrations.SubscribeToPostgres warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210729161959_subscribe_to_postgres.exs:21: Logflare.Repo.Migrations.SubscribeToPostgres warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210729161959_subscribe_to_postgres.exs:22: Logflare.Repo.Migrations.SubscribeToPostgres warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20210729161959_subscribe_to_postgres.exs:23: Logflare.Repo.Migrations.SubscribeToPostgres warning: an expression is always required on the right side of ->. Please provide a value after -> /opt/app/rel/logflare/lib/logflare-1.4.0/priv/repo/migrations/20230727111150_update_endpoint_query_language_de                     service not healthy: [supabase_analytics_tracecat] Try rerunning the command with --debug to troubleshoot the error.
Error encountered while starting Supabase:
Try rerunning the command with --debug to troubleshoot the error.
```

**To Reproduce**

...

**Expected behavior**

Installation should work. Getting started with the project is so difficult. I am slowly losing my will to try tracecat...

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu

**Additional context**
I am using a static ngrok url with port 8001. Idk if this helps.

After every fail run  the script can't get the secret key from supabase service. Thats why i delete all docker volumes of the project and all folders, before i start again. Hope this ensures to not get any bugs regarding a conflict between new and old installation try

**Comment by topher-lo** [comment]
@Ohmsnipe we hear you. That's why we're moving off Supabase: https://github.com/TracecatHQ/tracecat/issues/95

This migration is our top priority: expect an update within the next week. We prioritized features over deployment and are fixing this friction now.

**Comment by Ohmsnipe** [comment]
> @Ohmsnipe we hear you. That's why we're moving off Supabase: #95
> 
> This migration is our top priority: expect an update within the next week. We prioritized features over deployment and are fixing this friction now.

Sounds promising. Thank you!

**Comment by daryllimyt** [comment]
Closed by #106
